test/validation/api/traffic_mngr/traffic_mngr.c

```diff
@@ -2451,6 +2451,14 @@ static int traffic_mngr_check_shaper(void)
 	odp_cpumask_t cpumask;
 	int cpucount = odp_cpumask_all_available(&cpumask);
 
+/* Skip the shaper test on arm64 systems */
+#if defined(__aarch64__)
+	printf("\nTemporarily skip shaper test which intermittenly "
```

Comment:
Typo: intermittenly -> intermittently. The alignment of the following two lines is also a bit questionable but checkpatch seems to be ok with it.

Reply:
I will make these changes. Thanks Matias!